crates/bifrost/benches/append_throughput.rs

```diff
@@ -0,0 +1,235 @@
+// Copyright (c) 2024 -  Restate Software, Inc., Restate GmbH.
+// All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use std::ops::Range;
+
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion, Throughput};
+use futures::stream::{FuturesOrdered, FuturesUnordered};
+use futures::StreamExt;
+use restate_bifrost::{Bifrost, BifrostService};
+use restate_core::metadata;
+use restate_rocksdb::{DbName, Owner, RocksDbManager};
+use restate_types::config::{
+    BifrostOptionsBuilder, CommonOptionsBuilder, ConfigurationBuilder, LocalLogletOptionsBuilder,
+};
+use restate_types::logs::{LogId, Payload};
+use tracing::info;
+use tracing_subscriber::EnvFilter;
+mod util;
+
+async fn append_records_multi_log(bifrost: Bifrost, log_id_range: Range<u64>, count_per_log: u64) {
+    // for every log there is N streams. total count_per_log / streams_per_log gives the number of
```

Comment:
What is `streams_per_log` referring to?

Reply:
An old iteration of this test, I'll remove.